.github/ISSUE_TEMPLATE/samples-issue.md

```diff
@@ -1,10 +1,7 @@
 ---
 name: Samples Issue
 about: Report a issue with the Uno samples
-labels:
-- kind/consumer-experience
-- kind/documentation
-- triage/untriaged
+labels: kind/consumer-experience, kind/documentation, triage/untriaged
```

Comment:
maybe `contributor-experience` instead of `consumer-experience`? The samples are really more intended to test and verify behaviour during Uno development, rather than consumer-facing usage examples. (The Playground/Uno Gallery is for that https://github.com/nventive/Uno.Playground)

Reply:
I agree. Leave it with me to change to `contributor-experience`